fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoBackend.java

```diff
@@ -1056,6 +1088,7 @@ synchronized PTabletWriterAddBatchResult tabletWriterAddChunk(PTabletWriterAddCh
                     result.status.statusCode = TStatusCode.INTERNAL_ERROR.getValue();
                     result.status.errorMsgs.add(e.getMessage());
                 }
+                updateDiskUsage(PseudoBackend.DEFAULT_CHUNK_SIZE_ON_DISK_B);
```

Comment:
looks like the disk usage is only increasing? what if write/publish failed or tablet is dropped?

Reply:
Yes, tablet drop should be considered, i will fix it. For write failure, the disk usage won't be increased, but if write succeeded and publish failed, this disk usage is not decreased, maybe we should `updateDiskUsage()` only when publish success, i.e. in `commitNextRowset()`？

Reply:
How about this flow: refresh tablet disk usage when tablet creation/commitNextRowset/clone/drop and gc(in future) -> calculate delta -> update global diskusage